src/include/configuration.h

```diff
@@ -182,6 +182,17 @@ pgagroal_reload_configuration(void);
 void
 pgagroal_init_pidfile_if_needed(struct configuration* config);
 
+/**
+ * Checks if the configuration has a min set of values to
+ * take into account doing a prefill. For example, there must
+ * be users and limits set, otherwise it does not
+ * make any sense to attempt a prefill.
+ * This can be used to wrap the condituion before calling
+ * other prefill functions, e.g., `pgagroal_prefill()`.
+ */
+bool
+pgagroal_can_prefill(struct configuration* config);
```

Comment:
You don't need `config` here -- just resolve it internally in the function

Reply:
uhm, I don't fully agree. Having the `config` parameter clearly indicates this function is tied "only" to the configuration, not to other run-time parameters (e.g., the server is reachable). Second, I think this could also be useful when doing a `reload` to choose among new servers what can be used as a prefill-able one.
By the way, if you believe it is better to have the function without the parameter, I will make the change as a separated commit, so you can choose.